advanced/myserverextension/myserverextension/_version.py

```diff
@@ -0,0 +1,2 @@
+version_info = (0, 0, 0)
```

Comment:
Should we start at `0.1.0`?

Reply:
Yes this makes more sense for consistency with the other example